docs/reference/index-modules/translog.asciidoc

```diff
@@ -29,13 +29,11 @@ The data in the translog is only persisted to disk when the translog is
 ++fsync++ed and committed.  In the event of hardware failure, any data written
 since the previous translog commit will be lost.
 
-By default, Elasticsearch ++fsync++s and commits the translog every 5 seconds
-if `index.translog.durability` is set to `async` or if set to `request`
-(default) at the end of every <<docs-index_,index>>, <<docs-delete,delete>>,
-<<docs-update,update>>, or  <<docs-bulk,bulk>> request. More precisely, if set
-to `request`, Elasticsearch will only report success of an index, delete,
+By default, Elasticsearch will only report success of an index, delete,
```

Comment:
It's perhaps a little unclear that the "by default" is talking about the default of the `index.translog.durability` setting. How about something like this?

```suggestion
By default, `index.translog.durability` is set to `request` meaning that Elasticsearch will only report success of an index, delete,
```